<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomMsgBox</class>
 <widget class="QDialog" name="CustomMsgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Внимание!</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_message">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_createLink">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_dontLink">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_mergeFirst">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_mergeSecond">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_abortSaving">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
